src/components/SelectArticle.jsx

```diff
@@ -1,59 +1,58 @@
 import { useParams } from "react-router-dom";
 import { useState, useEffect } from "react";
-import { getArticleById, getCommentsById } from "../utils/Api";
+import { getArticleById, getCommentsById, updateArticle } from "../utils/Api";
+import Comments from "./Comments";
 
 const SelectArticle = () => {
   const { article_id } = useParams();
   const [article, setArticle] = useState({});
   const [commentsById, setCommentsById] = useState({});
+  const [votes, setVotes] = useState(0)
 
   useEffect(() => {
     getArticleById(article_id).then((response) => {
+        console.log(response)
       setArticle(response);
-      
+        setVotes(response.votes)
     });
-    getCommentsById(article_id).then((response) => {
-        setCommentsById(response)
-      });
   }, [article_id]);
 
-// console.log(article)
-//  console.log(commentsById)
-
-//  Object.values(commentsById).map((comment) => {
+  useEffect(() => {
+    getCommentsById(article_id).then((response) => {
+      setCommentsById(response);
+    });
+  }, [article_id]);
```

Comment:
Both of your useEffects are tracking the article_id - could these be combined into one useEffect?